### ISSUE TITLE
navigation.tabs not working

### ISSUE THREAD

**Issue by memeplex** [squidfunk/mkdocs-material#2694]
- [x] I've read the [contribution guidelines][1] and agree with them

__I've found a bug and checked that ...__

- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... the documentation does not mention anything about my problem
- [x] ... there are no open or closed issues that are related to my problem

Indeed, there are related closed issues, as it's usually the case, but I found nothing helpful, seem to be mostly outdated.

## Description

I'm not able to enable a tab bar as described in https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-tabs.

### Expected behavior

Tabs will show if navigation.tabs is listed as a feature.

### Actual behavior

Nothing changes wrt to the default layout.

### Steps to reproduce the bug

1. In a temporary folder run:
```sh
mkdir docs
mkdir docs/eda
touch docs/index.md
touch docs/eda/index.md
touch docs/eda/eda.md
cat > mkdocs.yml <<EOF
site_name: public

theme:
  name: material
  features:
    - navigation.tabs

nav:
  - Main: index.md
  - Eda:
    - About: eda/index.md
    - Library: eda/eda.md
EOF

mkdocs serve
```

2. Open the site in your browser

### Package versions

* Python: 3.9.5
* MkDocs: 1.1.2
* Material: 7.1.5

### Project configuration

``` yaml
site_name: public

theme:
  name: material
  features:
    - navigation.tabs

nav:
  - Main: index.md
  - Eda:
    - About: eda/index.md
    - Library: eda/eda.md
```

### System information

* OS: MacOS 11.3.1
* Browser: Chrome 90.0.4430.212 (Official Build) (x86_64)

  [1]: https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md

**Comment by memeplex** [comment]
In my example above there are no proper top-level mardown sections in the (empty) files. Since the documentation states:

> When tabs are enabled, top-level sections are rendered in a menu layer below the header for viewports above 1220px, but remain as-is on mobile

I've added top level headers to each file but to no avail.

**Comment by memeplex** [comment]
Tabs are indeed written to the html output.


```html
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-tabs__inner md-grid">
```

but are set to display:none, I don't know why.

**Comment by memeplex** [comment]
Neither `navigation.sections` changes anything. I don't know, either there is a general issue related to the nav structure or, more probably, I'm not getting how it is supposed to work.

**Comment by squidfunk** [comment]
Thanks for reporting. What's your screen resolution?

**Comment by memeplex** [comment]
Ok I was able to test this in a bigger screen and indeed it works, then I changed the scale factor of my macbook (I prefer the old 2x default, looks like 1280 x 800, over the modern 1.5x) to 1.5x, and it also fixed the issue. Still, I believe my screen is large enough to show tabs, is there a way to force that?

**Comment by squidfunk** [comment]
The layout is computed in REM units based on a default font-size of 16px. If you use a scale factor of 1.5, 1440px become 960px (1440/ 1.5), which is smaller than the 1220px, the threshold at which tabs are shown. Thus, tabs are not shown and you're viewing the tablet landscape view.

Closing, as this is not an error with this theme. The theme actually does the right job - respect your default font size.

**Comment by memeplex** [comment]
Yes I agree, thanks, I was going to close it myself. But would you mind giving me a hint about how to tweak it a little in order to make it work in medium sized laptops as in larger screens, if that's not too difficult? I'm really out of my comfort zone here.

**Comment by squidfunk** [comment]
Sure, no problem. The changes would need to be made in the breakpoint configuration:

https://github.com/squidfunk/mkdocs-material/blob/1d3d3e6681570ad2b5b4ca416aa8f5d4fd2bb3e4/src/assets/stylesheets/_config.scss#L28-L42

You would need to recompile the theme. See the [customization guide](https://squidfunk.github.io/mkdocs-material/customization/#theme-development) how to do that. You might also need to adjust the breakpoints in the application source:

https://github.com/squidfunk/mkdocs-material/blob/1d3d3e6681570ad2b5b4ca416aa8f5d4fd2bb3e4/src/assets/javascripts/bundle.ts#L93-L94

Note that customization of breakpoints is nothing we're optimizing for or can offer any support for. You're essentially changing deep configuration settings that can be changed, so you're on your own. However, you're encouraged to create a discussion to ask other users who might have been successful to share their results.

**Comment by memeplex** [comment]
Great! Thank you very much!

I checked the specs of my MacBook Pro 13'' 2017, here is a more accurate description:

1. Screen height is 2560x1800 pixels. That's why in my 2x setting it says "looks like 1280 x 800".
2. The default resolution is 1.75x and "looks like 1440 x 900".
3. There is a "More space" resolution, which is microscopic for my taste, at 1.5x.

The old default of 2x avoided fractional scaling, which is why I prefer it, but a few years ago they settled on 1.75x as a fair compromise.

Given the numbers above, I wasn't explain the difference in behavior between the 1.75x and the 2x modes since both are above the 1220px threshold. Chrome is at 100% zoom... but then I realized my default font in Chrome is customized to 17px, so it's probably that which makes the difference. Changing it to 16 px (or setting a zoom factor of 90%) makes the tab bar visible even at 2x.

Thanks again!
